packages/react-sdk/src/db/style.ts

```diff
@@ -0,0 +1,49 @@
+import type { StyleProperty, StyleValue } from "@webstudio-is/css-data";
+import { SharedStyleValue } from "@webstudio-is/css-data";
+import { type ComponentName, getComponentMeta } from "../components";
+import { z } from "zod";
+
+export type PresetStyleItem = {
+  component: ComponentName;
+  property: StyleProperty;
+  value: StyleValue;
+};
+
+// @todo can't figure out how to make component and property to be enum
```

Comment:
@Andarist can you help here please?

Reply:
It's copy paste from instance type which has the same issue with component. Just didn't want to solve it here.

Reply:
I can look into it in the next couple of days. Right now I’m on vacation